### DISCUSSION TITLE
%E5%8F%91%E7%8E%B0%E5%8F%B7%20DIS/[STCN][%E6%98%9F%E9%99%85%E8%BF%B7%E8%88%AA_%E5%8F%91%E7%8E%B0%E5%8F%B7][S05][WEBRip_1080P][HEVC-10bit_DDP5.1_AAC]

### DISCUSSION THREAD

**Discussion by giscus** [narwhrl/narwhrl.github.io#16] [Giscus]
# %E5%8F%91%E7%8E%B0%E5%8F%B7%20DIS/[STCN][%E6%98%9F%E9%99%85%E8%BF%B7%E8%88%AA_%E5%8F%91%E7%8E%B0%E5%8F%B7][S05][WEBRip_1080P][HEVC-10bit_DDP5.1_AAC]

https://pan.startrekcn.cn/%E5%8F%91%E7%8E%B0%E5%8F%B7%20DIS/[STCN][%E6%98%9F%E9%99%85%E8%BF%B7%E8%88%AA_%E5%8F%91%E7%8E%B0%E5%8F%B7][S05][WEBRip_1080P][HEVC-10bit_DDP5.1_AAC]

**Comment by itsLittleKevin** [comment]
## *你们问我资瓷不资瓷，我是资瓷的*
